crates/turbopack-css/src/references/import.rs

```diff
@@ -186,5 +196,84 @@ impl ValueToString for ImportAssetReference {
     }
 }
 
+#[turbo_tasks::value_impl]
+impl CodeGenerateable for ImportAssetReference {
+    #[turbo_tasks::function]
+    async fn code_generation(
+        self_vc: ImportAssetReferenceVc,
+        _context: ChunkingContextVc,
+    ) -> Result<CodeGenerationVc> {
+        let reference = &*self_vc.await?;
+        let request = (*reference.request.await?).clone();
+        let visitors = match request {
+            Request::Uri {
+                protocol,
+                remainder,
+            } => {
+                vec![
+                    create_visitor!(visit_mut_stylesheet(stylesheet: &mut Stylesheet) {
+                        let mut last_import_index = None;
+                        println!("iterating stylesheet rules");
+                        for (i, rule) in stylesheet.rules.iter().enumerate() {
```

Comment:
This procedure of linearly scanning for the last import feels pretty suboptimal. I'll explore using the ast visitor to prepend the rule when we visit one that's not an import.